planner/core/testdata/plan_suite_in.json

```diff
@@ -1094,5 +1094,37 @@
       "SELECT ta.NAME FROM ta WHERE EXISTS (select /*+ no_decorrelate() */ 1 from tb where ta.code = tb.code and tb.NAME LIKE 'chad9%') AND (select /*+ no_decorrelate() */ max(id) from tc where ta.name=tc.name and tc.name like 'chad99%') > 100 and (select /*+ no_decorrelate() */ max(id) from td where ta.id=td.id and td.name like 'chad999%') > 100"
 
     ]
+  },
+  {
+    "name": "TestCountStarForTikv",
+    "cases": [
+      "select count(*) from t",
+      "select count(1), count(3.1415), count(0), count(null) from t -- be rewritten count(row_id) but count(null)",
```

Comment:
Comment should be updated for all the tikv tests, since the rule doesn't apply on tikv storage.

Reply:
We discussed this issue before. Although tikv still selects the rowid column, count(1) will be rewritten as count(row_id) when displayed. So what I mainly say in my comments is the rewriting of this part.
(Notes are written differently from tiflash.)